File: Python/01.ProgrammingBasics/04.Conditional-Statements-Exersise/05.Godzillavs.Kong/05.GodzillavsKong.py
budget = float(input())
statisticians_count = int(input())
clothing_price_per_statistician = float(input())

decor_price = budget * 0.10
clothing_price = statisticians_count * clothing_price_per_statistician

if statisticians_count > 150:
    clothing_price -= clothing_price * 0.10

total_cost = decor_price + clothing_price

if total_cost > budget:
    needed_money = total_cost - budget
    print("Not enough money!")
    print(f"Wingard needs {needed_money:.2f} leva more.")
else:
    money_left = budget - total_cost
    print("Action!")
    print(f"Wingard starts filming with {money_left:.2f} leva left.")

File: C++/01-Basics/02.3-ConditionalStatementsExercise/05.GodzillavsKong/05.GodzillavsKong.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    double budget, clothingPrice;
    int actors;

    cin >> budget >> actors >> clothingPrice;

    double decorPrice = budget * 0.10;
    double totalClothingPrice = actors * clothingPrice;

    if (actors > 150) {
        totalClothingPrice -= totalClothingPrice * 0.10;
    }

    double totalFilmPrice = decorPrice + totalClothingPrice;

    cout << fixed << setprecision(2);

    if (totalFilmPrice <= budget) {
        cout << "Action!" << endl;
        cout << "Wingard starts filming with " << budget - totalFilmPrice << " leva left." << endl;
    }
    else {
        cout << "Not enough money!" << endl;
        cout << "Wingard needs " << totalFilmPrice - budget << " leva more." << endl;
    }

    return 0;
}
